<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_maschinen</class>
 <widget class="QDialog" name="Dialog_maschinen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CNC-Maschinen</string>
  </property>
  <widget class="QListWidget" name="listWidget_maschinen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>251</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_wkz">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Werkzeug editieren</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>51</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_tischbreite"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_tischlaenge">
      <property name="text">
       <string>Tischlänge</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_tischbreite">
      <property name="text">
       <string>Tischbreite</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_tischlaenge"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_laenge_ok">
      <property name="text">
       <string>übernehmen</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_breite_ok">
      <property name="text">
       <string>übernehmen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>511</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_neue_maschine">
      <property name="text">
       <string>Neu</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_speierort">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>511</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Speicherort der Maschine</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_ausgabe">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Ausgabeformat</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_ausgabe_kein">
       <property name="text">
        <string>keine Ausgabe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_ausgabe_emc2">
       <property name="text">
        <string>LinuxCNC (emc2)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
